src/shared/components/TimeSeries.tsx

```diff
@@ -241,8 +239,7 @@ class TimeSeries extends Component<Props, State> {
 
       // Issue new queries
       this.pendingResults = queries.map(({text}) => {
-        const orgID =
-          getOrgIDFromBuckets(text, buckets) || this.props.match.params.orgID
+        const orgID = this.props.match.params.orgID
```

Comment:
Minor tweak, but I think this line can be pulled up to the top of the `reload` function, rather than being called on every iteration of the loop, since it doesn't need to use `text` anymore.

That's how the code was structured before this change was implemented. See [this line](https://github.com/influxdata/ui/commit/45d1b2d30a5f0481a5db56bb9a31e3d07cbf2e3b#diff-43db5ef9584cae7f3e2a4eceae743ecc2dda44e9b26fc68f467dffbd62bf637cL158).

If you don't know how I found that line, I did it like this: I opened `TimeSeries.tsx` in the github UI in a separate window, then clicked on on the icon that appears and clicking `view git blame for line`, then followed that line through history to see when it was initially introduced.

Reply:
Yeah, it makes sense to move the `orgID` up to the top. Thanks for the `view git blame for line` instruction! It will be helpful for future development!